haas/cli.py

```diff
@@ -401,6 +431,24 @@ def stop_console(node):
     do_delete(url)
 
 @cmd
+def make_initial_admin(username, password):
```

Comment:
Ya - I'd like `create_admin_user`, with documentation emphasizing that the `user_*` commands should be used after initial setup.